docs/developer-guide/working-with-plugins.md

```diff
@@ -18,6 +18,25 @@ module.exports = {
 };
 ```
 
+### Preprocessors in Plugins
+
+You can also create plugins that would tell ESLint how to process files other then JavaScript. In order to create a
+preprocessor, object that is exported from your module has to conform to the following interface:
+
+```js
+preprocessors: {
```

Comment:
The problem with "preprocessors" is that this is both a preprocessor and a postprocessor, so it's not accurate (and potentially confusing).